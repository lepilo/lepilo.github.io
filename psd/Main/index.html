<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Main</title>
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
<header>
	<div class="container">
		<div class="top clearfix">
			<div class="logo left box center-t">
				<p class="logo__text"><span class="red">S</span>mart</p>
			</div>
			<nav class="right">
				<ul class="menu">
					<li class="menu__item"><a href="#">some text 1</a></li>
					<li class="menu__item"><a href="#">some text 2</a></li>
					<li class="menu__item"><a href="#">some text 3</a></li>
					<li class="menu__item"><a href="#">some text 4</a></li>
				</ul>
			</nav>
		</div>
		<section class="header__text center-t">
			<h1 class="title"><span class="red">Smart</span> Agency Landign Page</h1>
			<p class="subtitle">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Maiores, fuga provident. Libero magni quibusdam labore ad iste saepe hic, maxime qui ipsum tempore beatae, volu tomi ptates &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;doloremque rerum aspernatur dolorum at!</p>
		</section>
		<div class="message center-t box">
			<h3 class="message__title"><span class="red">Send</span> A Message</h3>
			<form action="#">
				<input type="text" name="name" id="" placeholder="Your Name">
				<input type="email" name="email" id="" placeholder="Your Email">
				<textarea type="text" name="text" id="" placeholder="Your Message"></textarea>
			</form>
			<button class="message__button">Send Message</button>
		</div>
	</div>
</header>
<main class+"content">
	<section class="process center-t">
		<div class="container">
			<h1 class="title"><span class="red">Working</span> Process</h1>
			<p class="subtitle">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quisquam temporibus ipsum consectetur esse.</p>
			<div class="know clearfix">
				<div class="know__item box center-t left">
					<h4 class="know__title">Koc Koi Tomi</h4>
					<p class="know__text">Lorem ipsum dolor sit amet, consectetur.</p>
					<button class="know__button">Know More</button>
				</div>
				<div class="know__item know__item--second box center-t left">
					<h4 class="know__title">Koimona Set</h4>
					<p class="know__text">Lorem ipsum dolor sit amet, consectetur.</p>
					<button class="know__button">Know More</button>
				</div>
				<div class="know__item know__item--third box center-t left">
					<h4 class="know__title">Setting Koire</h4>
					<p class="know__text">Lorem ipsum dolor sit amet, consectetur.</p>
					<button class="know__button">Know More</button>
				</div>
				<div class="know__item know__item--four box center-t left">
					<h4 class="know__title">Ko Image Lara</h4>
					<p class="know__text">Lorem ipsum dolor sit amet, consectetur.</p>
					<button class="know__button">Know More</button>
				</div>
			</div>
		</div>
	</section>
	<section class="works  center-t">
		<div class="container">
			<h1 class="title"><span class="red">Featured</span> Works</h1>
			<p class="subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit Maiores fuga provident Libero.</p>
			<div class="grid clearfix">
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
				<div class="grid__items left"></div>
			</div>
		</div>
	</section>
	<section class="service center-t">
		<div class="container">
				<h1 class="title"><span class="red">Services</span> We Provide</h1>
				<p class="subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit Maiores fuga provident Libero.</p>
			<div class="service__left left-t">
				<h2 class="service__title">Modern App Design</h2>
				<p class="service__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt reiciendis veritatis fugit architecto eaque deleniti delectus, iusto. Quasi necessitatibus culpa temporibus mollitia commodi et similique delectus cum, debitis, distinctio nobis.</p>
				<p class="service__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis, officiis!</p>
				<p class="service__text">Lorem ipsum dolor sit amet.</p>
				<button class="service__button">Know More</button>
			</div>
			<div class="service__right">
				<img src="img/watch.png" alt="">
			</div>
		</div>
	</section>
</main>
	
</body>
</html>