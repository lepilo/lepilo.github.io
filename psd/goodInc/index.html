<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Good Inc.</title>
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<div class="wrapper">
		<header>
			<div class="container clearfix">
				<div class="logo left">
					<a href="#" class="logo__link">g</a>
					<span class="logo__span">Good Inc.</span>
				</div>
			</div>
		</header>
		<nav>
			<div class="container clearfix">
				<ul class="menu left">
					<li class="menu__item"><span class="menu__span"><a href="#" class="menu__link">Portfolio</a></span></li>
					<li class="menu__item"><span class="menu__span"><a href="#" class="menu__link">Services</a></span></li>
					<li class="menu__item"><span class="menu__span"><a href="#" class="menu__link">Staff</a></span></li>
					<li class="menu__item"><span class="menu__span"><a href="#" class="menu__link">Articles</a></span></li>
					<li class="menu__item"><span class="menu__span"><a href="#" class="menu__link">Contact</a></span></li>
				</ul>
			</div>	
		</nav>
		<div class="container">
			<div class="panel-header clearfix">
				<div class="search-wrap left">
					<form action="#">
						<input  class="search" type="text" placeholder="Search for somthing here">
					</form>
				</div>
			<div class="social right">
				<a href="#" class="social__link twit">Twitter</a>
				<a href="#" class="social__link face">Facebook</a>
				<a href="#" class="social__link rss">RSS</a>
				<a href="#" class="social__link mail">Mail</a>
			</div>
			</div>
			<div id="feature clearfix">
				<div class="feature__left left">
					<img src="img/feature__left.jpg" alt="" class="feature__img">
					<div class="feature__descr">
						<p class="feature__text">Good Inc. image caption to go here</p>
					</div>
				</div>
				<div class="feature__right left">
					<h4 class="feature__title">Housework can't kill you, but why take a chance?</h4>
					<p class="feature_r_text">Quisque accumsan eleifend dolor, nec bibendum nibh ultricies at. Nunc convallis risus sit amet augue bibendum non lobortis eros interdum. Maecenas quis risus felis, at volutpat elit...</p>
					<div class="text__right"><span class="date">14th April 2013</span></div>
				</div>
			</div>
			<section class="articles clearfix">
				<div class="articles__items left">
					<img src="img/artc1.jpg" alt="" class="articles__img">
					<h3 class="articles__titles"></h3>
					<p class="articles__text"></p>
				</div>
				<div class="articles__items left">
					<img src="img/arct2.jpg" alt="" class="articles__img">
					<h3 class="articles__titles"></h3>
					<p class="articles__text"></p>
				</div>
				<div class="articles__items left">
					<img src="img/artc3.jpg" alt="" class="articles__img">
					<h3 class="articles__titles"></h3>
					<p class="articles__text"></p>
				</div>
			</section>
		</div>
	</div>
</body>
</html>