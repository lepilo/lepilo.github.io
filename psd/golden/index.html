<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	
<div id="wrapper">
	<header id="header">
		<div class="container clearfix">
			<div class="logo left"><a href="#"><img src="img/logo.png" alt=""></a></div>
			<ul class="menu right">
				<li class="menu__item"><a href="#" class="menu__link">Home</a></li>
				<li class="menu__item"><a href="#" class="menu__link">Services</a></li>
				<li class="menu__item"><a href="#" class="menu__link">Portfolio</a></li>
				<li class="menu__item"><a href="#" class="menu__link">About</a></li>
				<li class="menu__item"><a href="#" class="menu__link">Contact</a></li>
			</ul>
		</div>
	</header>
	<section id="home">
		<div class="container">
			<h3 class="home__title1">Welcome To Our Studio!</h3>
			<h1 class="home__title2">Itâ€™s nice to Meet you</h1>
			<butt class="home__btn">Tell Me More</butt>
		</div>
	</section>
	<section id="services" class="box">
		<div class="container">
			<h2 class="services__title">Services</h2>
			<p class="services__descr">Proin iaculis purus consequat sem cure. </p>
			<div class="services-wrap clearfix">
				<div class="services__item left commerce">
					<h3 class="item__title">E-Commerce</h3>
					<p class="item__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
				</div>
				<div class="services__item left responsive">
					<h3 class="item__title">Responsive Web</h3>
					<p class="item__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
				</div>
				<div class="services__item left security">
					<h3 class="item__title">Web Security</h3>
					<p class="item__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio" class="box">
		<div class="container clearfix">
		<h2 class="services__title">our Portfolio</h2>
			<p class="services__descr">Proin iaculis purus consequat sem cure. </p>
			<div class="portfolio-wrap clearfix">
				<div class="portfolio__item left">
					<a href=""><img src="img/Image.jpg" alt=""></a>
					<div class="portfolio__footer">
						<h5 class="portfolio__title">Ebony & Ivory</h5>
						<span class="portfolio__subtitle">Branding</span>
					</div>
				</div>
				<div class="portfolio__item left">
					<a href=""><img src="img/Image1.jpg" alt=""></a>
					<div class="portfolio__footer">
						<h5 class="portfolio__title">Ebony & Ivory</h5>
						<span class="portfolio__subtitle">Branding</span>
					</div>
				</div>
				<div class="portfolio__item left">
					<a href=""><img src="img/Image2.jpg" alt=""></a>
					<div class="portfolio__footer">
						<h5 class="portfolio__title">Ebony & Ivory</h5>
						<span class="portfolio__subtitle">Branding</span>
					</div>
				</div>
				<div class="portfolio__item left">
					<a href=""><img src="img/Image3.jpg" alt=""></a>
					<div class="portfolio__footer">
						<h5 class="portfolio__title">Ebony & Ivory</h5>
						<span class="portfolio__subtitle">Branding</span>
					</div>
				</div>
				<div class="portfolio__item left">
					<a href=""><img src="img/Image4.jpg" alt=""></a>
					<div class="portfolio__footer">
						<h5 class="portfolio__title">Ebony & Ivory</h5>
						<span class="portfolio__subtitle">Branding</span>
					</div>
				</div>
				<div class="portfolio__item left">
					<a href=""><img src="img/Image5.jpg" alt=""></a>
					<div class="portfolio__footer">
						<h5 class="portfolio__title">Ebony & Ivory</h5>
						<span class="portfolio__subtitle">Branding</span>
					</div>
				</div>
			</div>
		</div>
	</section>
	<section id="about" class="box">
		<div class="container">
			<h1 class="about__title">about us</h1>
			<p class="about__subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="about-wrap">
				<div class="about__item clearfix">
					<div class="text left">
						<h5 class="text__title text-right">
							July 2010 <span class="text__sub">Our Humble Beginnings</span>
						</h5>
						<p class="text__article text-right">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
					</div>
					<div class="img left">
						<img src="img/Image7.png" alt="">
					</div>
				</div>
				<div class="about__item clearfix">
					<div class="text right">
						<h5 class="text__title text-left">
							July 2010 <span class="text__sub">Our Humble Beginnings</span>
						</h5>
						<p class="text__article text-left">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
					</div>
					<div class="img right">
						<img src="img/Image8.png" alt="">
					</div>
				</div>
				<div class="about__item clearfix">
					<div class="text left">
						<h5 class="text__title text-right">
							July 2010 <span class="text__sub">Our Humble Beginnings</span>
						</h5>
						<p class="text__article text-right">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
					</div>
					<div class="img left">
						<img src="img/Image9.png" alt="">
					</div>
				</div>
				<div class="about__item clearfix">
					<div class="text right">
						<h5 class="text__title text-left">
							July 2010 <span class="text__sub">Our Humble Beginnings</span>
						</h5>
						<p class="text__article text-left">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p>
					</div>
					<div class="img right">
						<img src="img/Image10.png" alt="">
					</div>
				</div>
				<a href="#" class="about__link">Our<span class="about__block">Story</span>
					<span class="about__block">Continues</span><span class="about__block">...</span>
				</a>
				
			</div>
		</div>	

	</section>
	<section id="team" class="box">
		<div class="container">
			<h2 class="services__title">our amazing team</h2>
			<p class="services__descr">Proin iaculis purus consequat sem cure. </p>
			<div class="team-wrap clearfix">
				<div class="team__item left">
					<div class="photo">
						<img src="" alt="" class="photo__img">
					</div>
					<p class="team__names">Kimberly Thompson</p>
					<span class="team__position">Marketer</span>
					<div class="social">
						<a href="#" class="tviter"></a>
						<a href="#" class="face"></a>
						<a href="#" class="pinterest"></a>
					</div>
				</div>
				<div class="team__item left">
					<div class="photo">
						<img src="" alt="" class="photo__img">
					</div>
					<p class="team__names">Rico Massimo</p>
					<span class="team__position">Coder</span>
					<div class="social social--center">
						<a href="#" class="tviter"></a>
						<a href="#" class="face"></a>
						<a href="#" class="google"></a>
					</div>
				</div>
				<div class="team__item left">
					<div class="photo">
						<img src="" alt="" class="photo__img">
					</div>
					<p class="team__names">Uku Mason</p>
					<span class="team__position">Graphic Designer</span>
					<div class=" social social--right">
						<a href="#" class="face"></a>
						<a href="#" class="pinterest"></a>
					</div>
				</div>
			</div>
			<p class="services__descr mod">Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit <span class="block">aenean rhoncus posuere odio in tincidunt proin iaculis.</span>
			</p>
		</div>
	</section>
	<section class="logo">
		<div class="container">
			<img src="img/Envato.jpg" alt="">
		</div>
	</section>
	<section id="contacts" class="box">
		<div class="container">
			<h2 class="services__title">contact us</h2>
			<p class="services__descr">Proin iaculis purus consequat sem cure. </p>
			<form action="" class="clearfix">
				<span class="info">
				<input type="text" name="name" id="" placeholder="Your Name *">
				<input type="text" name="email" id="" placeholder="Your E-mail *">
				<input type="text" name="subject" id="" placeholder="Subject">
				</span>
				<span class="block">
				<textarea name="" id="" cols="30" rows="10"></textarea>
				</span>
			</form>
			<a href="#" class="contacts__btn">Send Message</a>
		</div>
	</section>
	<footer id="footer">
		<div class="container clearfix">
			<p class="copyright left">&copy; Copyright 2014 FreebiesXpress.com</p>
			<div class="social social--footer">
						<a href="#" class="tviter"></a>
						<a href="#" class="face"></a>
						<a href="#" class="pinterest"></a>
						<a href="#" class="google"></a>
			</div>
		</div>
	</footer>
</div>

</body>
</html>